glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -305,6 +304,30 @@ open class GleanInternalAPI internal constructor () {
         }
     }
 
+    /**
+     * Send a list of pings.
+     *
+     * The ping content is assembled as soon as possible, but upload is not
+     * guaranteed to happen immediately, as that depends on the upload
+     * policies.
+     *
+     * If the ping currently contains no content, it will not be sent.
+     *
+     * @param pings List of pings to send.
+     * @return The async Job performing the work of assembling the ping
```

Comment:
Is `Job` here uppercased because that's the actual type of what `sendPingsGeneric` returns?

Reply:
Yes.  Should probably make this `[Job]` (which will link in the generated docs).